gemini/function-calling/sql-talk-app/app.py

```diff
@@ -0,0 +1,257 @@
+import time
+import streamlit as st
+from google.cloud import bigquery
+from vertexai.preview.generative_models import (
+    FunctionDeclaration,
+    GenerativeModel,
+    Part,
+    Tool,
+)
+
+sql_query_func = FunctionDeclaration(
+    name="sql_query",
+    description="Get information from data in BigQuery using SQL queries",
+    parameters={
```

Comment:
nit: Is there a class that can be used for `parameters`?

Reply:
Refer to [other comment](https://github.com/GoogleCloudPlatform/generative-ai/pull/385#discussion_r1490152681).